plugins/TodayAtMun/today.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime, timedelta
+from json import load
+from pathlib import Path
+from typing import Dict
+
+
+class Today:
+    """A class used to go find significant days on Mun Calendar"""
+
+    def __init__(self, diary: Dict):
+        self.diary = diary
+
+    def set_current_date(self) -> None:
+        """Current day, month, hour, second"""
+        self.date = datetime.now()
+
+    def get_current_date(self) -> datetime:
+        """Getter method for date"""
```

Comment:
Docstring describes what the function actually is (a getter method), rather than what it does (retrieves the currently set date). Additionally, missing trailing period.